data_gateway/packet_reader.py

```diff
@@ -1,4 +1,4 @@
-import datetime
+Startimport datetime

```

Comment:
There's an accidental "Start" added to the start of this line causing the tests to fail